app/src/main/java/com/graeberj/taskman/auth/presentation/registration/RegistrationState.kt

```diff
@@ -4,10 +4,11 @@ data class RegistrationState(
     val username: String = "",
     val email: String = "",
     val password: String = "",
-    val usernameError: Boolean = false,
+    val showUsernameError: Boolean = false,
     val showEmailError: Boolean = false,
-    val passwordError: Boolean = false,
+    val showPasswordError: Boolean = false,
```

Comment:
I assume, you also want to show some form of error message below the text, right?
In that case, you can work with one field `passwordError: String?` which contains both the information of the boolean as well as the error message. `passwordError != null` means there is an error and then you can directly get the message